### PR TITLE
fix(config-manager): Updated polling interval condition.

### PR DESCRIPTION
**Pull request by rashidsp**
Summary
-------

-  Update condition for using default update_interval when polling files.

### CODE REVIEW

**Comment by msohailhussain** [comment]
Please update readme.md for valid polling interval.

**Comment by msohailhussain** [comment]
Please update readme.md for polling interval.

File: optimizely/config_manager.py [hunk 1]
```diff
@@ -246,8 +246,8 @@ def set_update_interval(self, update_interval):
                 'Invalid update_interval "{}" provided.'.format(update_interval)
             )
 
-        # If polling interval is less than minimum allowed interval then set it to default update interval.
-        if update_interval < enums.ConfigManager.MIN_UPDATE_INTERVAL:
+        # If polling interval is less than or equal to 0 then set it to default update interval.
+        if update_interval <= 0:
             self.logger.debug('update_interval value {} too small. Defaulting to {}'.format(
```
Comment (msohailhussain) [inline optimizely/config_manager.py]: `update_interval value is not greater than zero, Defaulting to {} value`. @aliabbasrizvi WDYT?

Comment (aliabbasrizvi) [reply optimizely/config_manager.py]: I think current message is fine.

**Pull request merged**